core-bundle/src/Routing/RouteProvider.php

```diff
@@ -533,7 +539,7 @@ private function findRootPages(string $httpHost): array
             $rootPages = $pages->getModels();
         }
 
-        $pages = $pageModel->findBy(["tl_page.alias='index' OR tl_page.alias='/'"], null);
+        $pages = $pageModel->findBy(["tl_page.alias IN ('index', '/')"], null);
```

Comment:
What is the reasoning behind this change?

Reply:
I _thought_ mysql can use the index for this query, because we're no longer using `OR`. But I'm not sure that's really true.

Reply:
I’m pretty sure that MySQL handles both variants the same way.

Reply:
so should I revert this?